docs/source/yelpsoa_configs.rst

```diff
@@ -609,6 +609,17 @@ Each Tron **action** of a job MAY specify the following:
     then the command should be something available in the docker container (it should NOT
     start with ``paasta local-run``).
 
+If a Tron **action** of a job is of executor type ``spark``, it MAY specify the following:
+
+  * ``spark_cluster``: The Paasta cluster on which to run spark jobs. Default to the same
+    cluster the tron job is running on.
+
+  * ``spark_pool``: The Paasta pool on which to run spark jobs. Default to ``default`` pool
```

Comment:
Can you make this default to run on the same pool as the job itself? It would be nice if users didn't have to specify this twice.

Reply:
Is this the pool/cluster that the driver will run on or that the executors will run on?  The text makes it a little unclear.

If it's the executors, we definitely don't want to default them to the `default` pool.  If it's the driver, I don't think we want to default it to whatever the executors are running on.  The executors should run on a `batch` or `stable_batch` pool, and unless I'm mistaken we haven't changed the metrics we send to clusterman to include the driver's resources.

Reply:
This is the executors.  I am hesistant to have it default to run on the same pool as the job itself, because the job itself more than likely be running on a different cluster entirely (and probably running on a `batch` pool).  I agree the `default` pool probably isn't the right default, maybe just have it run on the `batch` pool by default?